github/enum/Modals.ts

```diff
@@ -38,116 +38,116 @@ export enum ModalsEnum {
-    PULL_REQUEST_COMMIT_MESSAGE_INPUT="pull-request-commit-message-input",
-    PULL_REQUEST_COMMIT_MESSAGE_LABEL="Merge Commit Message",
-    PULL_REQUEST_COMMIT_MESSAGE_ACTION="pull-request-commit-message-action",
-    PULL_REQUEST_COMMIT_MESSAGE_PLACEHOLDER="The New Pull Request Component Adds...",
-    PULL_REQUEST_MERGE_METHOD_OPTION="pull-request-merge-method-option",
-    PULL_REQUEST_MERGE_METHOD_INPUT_LABEL="Merge Method",
-    PULL_REQUEST_MERGE_METHOD_INPUT="pull-request-merge-method-input",
-    MESSAGE_MODAL_VIEW="message-modal-view",
-    ADD_PULL_REQUEST_COMMENT_VIEW_TITLE="GitHub Comments",
-    ADD_PULL_REQUEST_COMMENT_VIEW="add-pull-request-comment-view",
-    PULL_REQUEST_COMMENT_INPUT="pull-request-comment-input",
-    PULL_REQUEST_COMMENT_INPUT_LABEL="Enter Comment",
-    PULL_REQUEST_COMMENT_INPUT_PLACEHOLDER="@RonLek can you please reveiw the PR ?",
-    PULL_REQUEST_COMMENT_INPUT_ACTION="add-pull-request-comment-input-actions",
-    PULL_REQUEST_COMMENT_VIEW_TITLE="GitHub Comments",
-    PULL_REQUEST_COMMENTS_MODAL_VIEW="pull-request-commments-list-view",
-    PR_COMMENT_LIST_ACTION="pull-request-comment-list-action",
-    PR_COMMENT_LIST_LABEL="Comments",
-    ISSUE_NUMBER_LABEL="Issue Number",
-    ISSUE_COMMENT_LIST_ACTION="issues-comment-list-action",
-    ISSUE_COMMENT_LIST_LABEL="Comments",
-    ISSUE_COMMENTS_MODAL_VIEW="issue-commments-list-view",
-    ADD_ISSUE_COMMENT_VIEW="add-issue-comment-view",
-    ISSUE_COMMENT_INPUT="issue-comment-input",
-    ISSUE_COMMENT_INPUT_LABEL="Enter Comment",
-    ISSUE_COMMENT_INPUT_PLACEHOLDER="@RonLek can you please assign this issue to me ?",
-    ISSUE_COMMENT_INPUT_ACTION="add-issue-comment-input-actions",
-    ISSUE_COMMENT_VIEW_TITLE="GitHub Comments",
-    ADD_ISSUE_COMMENT_VIEW_TITLE="GitHub Comments",
-    VIEW_USER_ACTION="view-user-action",
-    VIEW_USER_LABEL="User Profile",
-    SEARCH_VIEW="search-view",
-    SEARCH_VIEW_TITLE="GitHub Search",
+    OPEN_ADD_SUBSCRIPTIONS_MODAL = 'open-add-subscriptions',
+    OPEN_UPDATE_SUBSCRIPTIONS_MODAL = 'open-update-subscriptions',
+    OPEN_DELETE_SUBSCRIPTIONS_MODAL = 'open-update-subscriptions',
+    OPEN_ADD_SUBSCRIPTIONS_LABEL = 'Add Subscriptions',
+    OPEN_UPDATE_SUBSCRIPTIONS_LABEL = 'Update Subscriptions',
+    OPEN_DELETE_SUBSCRIPTIONS_LABEL = 'Delete Subscriptions',
+    DELETE_SUBSCRIPTION_ACTION = 'delete-subscription',
+    DELETE_SUBSCRIPTION_LABEL = 'Unsubscribe',
+    ADD_SUBSCRIPTION_ACTION = 'add-subscription',
+    ADD_SUBSCRIPTION_LABEL = 'Subscribe',
+    UPDATE_SUBSCRIPTION_ACTION = 'update-subscription',
+    UPDATE_SUBSCRIPTION_LABEL = 'Update',
+    DELETE_SUBSCIPTIONS_TITLE = 'Unsubscribe to Repositories',
+    ADD_SUBSCIPTIONS_TITLE = 'New Subscription',
+    OPEN_REPO_ACTION = 'open-repo',
+    OPEN_REPO_LABEL = 'Open',
+    REPO_NAME_INPUT = 'repo-name-input',
+    REPO_NAME_LABEL = 'Enter Full Repository Name',
+    REPO_NAME_PLACEHOLDER = 'Owner/Repository',
```

Comment:
@VipinDevelops  All these changes look unrelated to your PR. Looks like you are using a linter which is making changes as you save. This is bloating the PR. Please Fix this

Reply:
Sure I thought Its good to have formating in code , Removed :+1: